ansible_runner/runner.py

```diff
@@ -134,18 +134,29 @@ def run(self):
         }
 
         self.status_callback('running')
-        child = pexpect.spawn(
-            self.config.command[0],
-            self.config.command[1:],
-            cwd=self.config.cwd,
-            env=env,
-            ignore_sighup=True,
-            encoding='utf-8',
-            echo=False,
-            use_poll=self.config.pexpect_use_poll,
-        )
-        child.logfile_read = stdout_handle
         self.last_stdout_update = time.time()
+        try:
+            child = pexpect.spawn(
+                self.config.command[0],
+                self.config.command[1:],
+                cwd=self.config.cwd,
+                env=env,
+                ignore_sighup=True,
+                encoding='utf-8',
+                echo=False,
+                use_poll=self.config.pexpect_use_poll,
+            )
+            child.logfile_read = stdout_handle
+        except pexpect.exceptions.ExceptionPexpect as e:
+            child = collections.namedtuple(
```

Comment:
`pexpect` looks for `self.config.command[0]` on the system path, and if it's not, found this error is thrown (and the fork fails).

Reply:
This is what things like after my patch:

```shell
[root@isolated 47]# pwd
/tmp/awx_47_17x1e6qx/artifacts/47
[root@isolated 47]# ls -la
total 28
drwx------. 3 root root 4096 Apr  5 02:04 .
drwx------. 3 root root 4096 Apr  5 02:04 ..
-rw-------. 1 root root 2662 Apr  5 02:04 command
drwx------. 2 root root 4096 Apr  5 02:04 job_events
-rw-------. 1 root root    3 Apr  5 02:04 rc
-rw-------. 1 root root    6 Apr  5 02:04 status
-rw-------. 1 root root   58 Apr  5 02:04 stdout
[root@isolated 47]# cat stdout
The command was not found or was not executable: bwrap.
[root@isolated 47]# cat rc
127
[root@isolated 47]# cat status
failed
[root@isolated 47]# cat job_events/* | python -m json.tool
{
    "counter": 1,
    "end_line": 1,
    "event": "verbose",
    "runner_ident": "47",
    "start_line": 0,
    "stdout": "The command was not found or was not executable: bwrap",
    "uuid": "eedfa39c-cd08-408f-bf76-0629c5d0295d"
}
```

Here's what it looks like in AWX:

Reply:
<img width="1053" alt="image" src="https://user-images.githubusercontent.com/214912/55599533-49cc2480-5726-11e9-8e1b-8ebbaed93112.png">

Reply:
With process isolation disabled:

<img width="1117" alt="image" src="https://user-images.githubusercontent.com/214912/55599606-91eb4700-5726-11e9-935c-f450c92b4c0b.png">